package.json

```diff
@@ -126,6 +127,7 @@
     "uint8array-json-parser": "0.0.1",
     "uuid": "^8.1.0",
     "vega": "^5.21.0",
+    "vega-lite": "^4.17.0",
```

Comment:
Why is this one necessary?

Reply:
It was before in packages but only in the .lock file and not here. It is a dependency on react-vega.

Reply:
but what happens if you don't include this here? the .lock file should include dependencies for react-vega on its own so we don't need to, right?

Reply:
It needs to be installed. I think before it was installed but removed from the package.json but since we do npm ci and not  install, this wasnt removing it.